soh/soh/Enhancements/randomizer/3drando/location_access/locacc_gerudo_valley.cpp

```diff
@@ -80,7 +80,10 @@ void AreaTable_Init_GerudoValley() {
                   Entrance(GV_FORTRESS_SIDE, {[]{return true;}}),
   });
 
-  areaTable[GV_OCTOROK_GROTTO] = Area("GV Octorok Grotto", "GV Octorok Grotto", NONE, NO_DAY_NIGHT_CYCLE, {}, {}, {
+  areaTable[GV_OCTOROK_GROTTO] = Area("GV Octorok Grotto", "GV Octorok Grotto", NONE, NO_DAY_NIGHT_CYCLE, {}, {
+                  //Locations
+                  LocationAccess(GV_STORMS_GROTTO_BEEHIVE, {[]{return CanBreakBeehives;}}),
```

Comment:
I think this is in the wrong spot, Octorok Grotto and Storms Grotto are not the same thing. Should be one entry down.

Reply:
brain fart (fixed)